examples/pmdarima/train.py

```diff
@@ -0,0 +1,57 @@
+import mlflow
```

Comment:
This takes quite awhile to run (several minutes). Can we make the workload smaller and add some print statement progress indicators prior to the training portion (e.g. "constructing model...")?

Reply:
reduced the runtime to ~ 5 seconds and added additional status messages.